styleguide.md

```diff
@@ -7,7 +7,7 @@
 
 - Primary color: #1B73E8
 - Hover color: #2f78d8
-- Background color: ##fafafa
```

Comment:
hover state colors are not clear. check the color I sent in your Slack DM

Reply:
Will change them